src/web/app/pages-student/student-home-page/student-home-page.component.ts

```diff
@@ -96,12 +144,34 @@ export class StudentHomePageComponent implements OnInit {
             + 'Please refresh this page in a few minutes to see the course ' + `${this.recentlyJoinedCourseId}`
             + ' in the list below.');
       }
-
-    }, (resp: ErrorMessageOutput) => {
-      this.statusMessageService.showErrorMessage(resp.error.message);
+    }, (e: ErrorMessageOutput) => {
+      this.statusMessageService.showErrorMessage(e.error.message);
     });
   }
 
+  /**
+   * Checks if feedback session is opened.
+   */
+  isOpened(fs: FeedbackSession): boolean {
```

Comment:
Information like this is embedded in `session.submissionStatus` or `session.publishStatus`